commands/operator-sdk/cmd/up/local.go

```diff
@@ -142,23 +142,22 @@ func upLocalAnsible() {
 		KubeConfig: mgr.GetConfig(),
 	})
 	if err != nil {
-		logrus.Fatalf("error starting proxy: %v", err)
+		log.Fatalf("error starting proxy: (%v)", err)
 	}
 
 	// start the operator
 	go ansibleOperator.Run(done, mgr, "./"+ansibleScaffold.WatchesYamlFile, time.Minute)
 
 	// wait for either to finish
 	err = <-done
-	if err == nil {
-		logrus.Info("Exiting")
-	} else {
-		logrus.Fatal(err.Error())
+	if err != nil {
+		log.Fatal(err)
 	}
+	log.Info("Ansible operator started succesfully. Exiting.")
```

Comment:
@shawn-hurley @mhrivnak is this message correct? I want to add more context to this log message.

Reply:
No. Line 152 will block for as long as the operator is running. When the operator exits, either due to an error or a signal handler, in the usual case it will be long after startup. I suggest sticking with just "Exiting". We could try to add a reason for why it's exiting, but if `err` is `nil`, there isn't much to go on.